src/github.com/getlantern/peerscanner/host.go

```diff
@@ -407,11 +484,32 @@ func (h *host) isAbleToProxy() (bool, bool, error) {
 }
 
 func (h *host) doIsAbleToProxy() (bool, bool, error) {
+	if h.port == "" {
```

Comment:
This could use a comment. This is to support old clients that don't report the port, is that correct? So in that case we try both 80 and 443?